src/components/InstancesList/InstancesList.js

```diff
@@ -594,6 +623,53 @@ class InstancesList extends React.Component {
       };
     };
 
+    const setSortedColumn = (event) => {
+      const {
+        match: { path },
+        goTo,
+        getParams,
+      } = this.props;
+
+      onToggle();
+
+      const params = getParams();
+      params.sort = event.target.value;
+
+      const segmentsSortBy = this.state.segmentsSortBy.map((key) => {
```

Comment:
This is a duplicate of code from `didUpdate` - please create a separate method and call it in both places